docs/review_process.md

```diff
@@ -0,0 +1,19 @@
+# Review Process
```

Comment:
Ich würde mich vielleicht schon jetzt zu Beginn für eine Sprache entscheiden. :)

Reply:
Meinst du damit die Sprache die der User sieht ? Also z.B. auch Englisch ?

Reply:
Also klar kann man es auch auf die Sprache des nutzers ausdehnen, aber hier meinte ich jetzt vorwiegend die Dokumentationssprache im repo :)

Reply:
Achso, weil die README in deutsch geschrieben ist. Hast recht 😄  werde sie kurz übersetzen.